filebeat/filebeat.reference.yml

```diff
@@ -57,6 +57,18 @@ filebeat.modules:
     # can be added under this section.
     #prospector:
 
+  # SSL Request logs
+  #ssl_request:
```

Comment:
It seems the reference config and the short one are not in sync. Is it `ssl` or `ssl_request`?